src/hotspot/share/opto/escape.cpp

```diff
@@ -369,6 +369,18 @@ bool ConnectionGraph::compute_escape() {
       assert(ptn->escape_state() == PointsToNode::NoEscape && ptn->scalar_replaceable(), "sanity");
     }
   }
+
+  if (VerifyReduceAllocationMerges) {
```

Comment:
Add `#ifdef ASSERT`

Reply:
This block is already inside an `#ifdef ASSERT`.